.editorconfig

```diff
@@ -233,32 +231,34 @@ dotnet_naming_style.first_upper.capitalization = first_word_upper
 # prefix_interface_interface_with_i - Interfaces must be PascalCase and the first character of an interface must be an 'I'
 dotnet_naming_style.prefix_interface_interface_with_i.capitalization = pascal_case
 dotnet_naming_style.prefix_interface_interface_with_i.required_prefix = I
+# disallowed_by_design_guidelines - use to auto-format items that are disallowed by the design guidelines
+dotnet_naming_style.prefix_with_underscore.capitalization = pascal_case
+dotnet_naming_style.prefix_with_underscore.required_prefix = _
+# disallowed_by_design_guidelines - use to auto-format items that are disallowed by the design guidelines
+dotnet_naming_style.disallowed_by_design_guidelines.capitalization = pascal_case
```

Comment:
Been a while since I learned this but it looks like you have not defined the `disallowed_by_design_guidelines` symbol.

Setting a kind of error message using the prefix/suffix is a really interesting idea. Without the symbol, I'm not clear on where this is being applied.

Reply:
Rules are applied in order, and only the first match for a symbol is applied.
The rules for fields here are:
1. private_fields_rule
2. public_constant_fields_rule
3. public_static_readonly_fields_rule
4. match_all_fields_rule

Thus, if a field is private, it will only have the first rule applied.
If a field is a public constant, then it will have the second rule applied.
If a field is a public static readonly, then it will have the third rule applied.

Otherwise, the field is disallowed by the .NET guidelines, and the fourth rule applies.